esmpkg/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "esmpkg",
+  "version": "0.0.1",
+  "publishConfig": {
+    "access": "public",
+    "directory": "pkg"
+  },
+  "scripts": {
+    "build": "pika build",
+    "version": "pika build"
```

Comment:
that doesn't look right?

Reply:
I think this makes sense; it’s a hook that happens before publishing to npm making sure a fresh bulls is prepared (i.e. it’s not stale, from a previous commit or different branch)

Reply:
the "version" script is a hook that happens before publishing? Are you sure?

Reply:
Not technically it doesn’t, but in our workflow since we use Lerna, the versioning happens before publishing (I think? Please correct me if I’m still mistaken!).